docs/contributing/contribute.md

```diff
@@ -25,121 +25,4 @@ The goal of the community is to develop a cloud native edge computing platform b
-KubeEdge generally follows the standard [github pull request](https://help.github.com/articles/about-pull-requests/) process.
-
-In addition to the above process, a bot will begin applying structured labels to your PR.
-
-The bot may also make some helpful suggestions for commands to run in your PR to facilitate review.
-These `/command` options can be entered in comments to trigger auto-labeling and notifications.
-Refer to its [command reference documentation](https://go.k8s.io/bot-commands).
-
-### Code Review
-
-To make it easier for your PR to receive reviews, consider the reviewers will need you to:
-
-* follow [good coding guidelines](https://github.com/golang/go/wiki/CodeReviewComments).
-* write [good commit messages](https://chris.beams.io/posts/git-commit/).
-* break large changes into a logical series of smaller patches which individually make easily understandable changes, and in aggregate solve a broader issue.
-* label PRs with appropriate reviewers: to do this read the messages the bot sends you to guide you through the PR process.
-
-### Format of the commit message
-
-We follow a rough convention for commit messages that is designed to answer two questions: what changed and why.
-The subject line should feature the what and the body of the commit should describe the why.
-
-```
-scripts: add test codes for metamanager
-
-this add some unit test codes to improve code coverage for metamanager
-
-Fixes #12
-```
-
-The format can be described more formally as follows:
-
-```
-<subsystem>: <what changed>
-<BLANK LINE>
-<why this change was made>
-<BLANK LINE>
-<footer>
-```
-
-The first line is the subject and should be no longer than 70 characters, the second line is always blank, and other lines should be wrapped at 80 characters. This allows the message to be easier to read on GitHub as well as in various git tools.
-
-Note: if your pull request isn't getting enough attention, you can use the reach out on Slack to get help finding reviewers.
-
-### Testing
-
-There are multiple types of tests.
-The location of the test code varies with type, as do the specifics of the environment needed to successfully run the test:
-
-* Unit: These confirm that a particular function behaves as intended. Unit test source code can be found adjacent to the corresponding source code within a given package. These are easily run locally by any developer.
-* Integration: These tests cover interactions of package components or interactions between KubeEdge components and Kubernetes control plane components like API server.  An example would be testing whether the device controller is able to create config maps when device CRDs are created in the API server.
-* End-to-end ("e2e"): These are broad tests of overall system behavior and coherence. The e2e tests are in [kubeedge e2e](https://github.com/kubeedge/kubeedge/tree/master/tests/e2e).
-
-Continuous integration will run these tests on PRs.
+- Read [this](../../CONTRIBUTING.md) for more details
```

Comment:
I would hope we can combine two doc into one.

Reply:
It should not be combined for now, since our website only cover docs under `docs`, it does not support soft link, and we need `CONTRIBUTING.md` under root dir.
It may be conbined after we use hugo to rebuild our website.

Reply:
why need CONTRIBUTING.md under root dir? cannot this move to docs as well?

Reply:
I heard it from @kevin-wangzefeng

Reply:
Get you!